DataFormats/FTLRecHit/src/classes_def.xml

```diff
@@ -1,5 +1,7 @@
 <lcgdict>
-  <class name="FTLUncalibratedRecHit" ClassVersion="3">
+  <class name="FTLUncalibratedRecHit" ClassVersion="5">
+    <version ClassVersion="5" checksum="508372968"/>
+    <version ClassVersion="4" checksum="2882178728"/>
```

Comment:
please drop the intermediate version and increment just by 1.
```xml
  <class name="FTLUncalibratedRecHit" ClassVersion="4">
    <version ClassVersion="4" checksum="508372968"/>
```

Reply:
Done.